packages/iframe-execution-environment-service/src/IframeExecutionEnvironmentService.test.ts

```diff
@@ -66,6 +67,7 @@ describe('Iframe Controller', () => {
     await expect(action()).rejects.toThrow(
       /Error while running plugin 'TestPlugin'/u,
     );
+    await iframeExecutionEnvironmentService.terminateAllPlugins();
```

Comment:
added `terminateAllPlugins` here because when testing it wasn't cleaning up the plugin from the page